README.md

```diff
@@ -126,9 +199,32 @@ If the argument you provide when calling the function holds values [a,b,c...z],
 Make sure to handle two special cases: length 0 (then return _true_) and length 1 (then return that single value).
 Again, don't use &&.
 
+function andN(values) {
+ var n = values.length;
+ if (n == 0) {
+ return true;
+ }
+ if (n == 1) {
+ return values[0]
+ }
+ for (var i = 0; i < n; i++;) {
+ if (!values[i]) {
+ return values[i];
+ }
+ }
+ return values[n-1];
+
 **d)**
 You've just realized that your effort was doomed: neither of your functions can replace the && operator in certain circumstances.  Explain why, and find an example which demonstrates failure.
 
+&& is sometimes the most complete way to tell the computer what conditions we want to meet. If/else statements work for basic problems, but if you're writing a really complex problem, conditional statements leave a lot of room for error and require much more computing power. && is fast and efficient and tells the computer that in order for condition 1 to be true, we MUST also meet conditions 2, 3, 4, etc.
+
+For example:
+console.log(eatFood && notDie);
+The only way to not die, is to not eat food.
+
```

Comment:
Right on! Also, it's good to know that a function, unlike the && operator, evaluates all of the arguments at once before it runs, whereas the && operand evaluates all of the arguments separately.